tests/.fmf/version

```diff
@@ -0,0 +1 @@
+1
```

Comment:
I'd really prefer to keep all Fedora related files in `tools/fedora`.

Reply:
This is a tricky one. Basically in the `fmf` tree I can only see and access files under the `.fmf` folder, e.g. if I have the file in `tests/.fmf/version` I can only access paths with `tests/**`. So from the `tools/fedora/.fmf/version`, I cannot access files in `tests/**`. There is an issue upstream about it [^1], but it's not a priority because there are workaround for it.

One workaround is if it's ok to put a `.fmf/version` in root, then I can move all of the test scripts to `tools/fedora` folder

[^1]: https://github.com/teemtee/tmt/issues/2568

Reply:
Could we maybe use something like the following?
```
tf_post_install_script: "mkdir .fmf && echo 1 > .fmf/version"
```

Reply:
> tf_post_install_script - Bash script as a string to run during the guest provisioning. 

I don't think so. The provisioning takes the `tools/fedora/plans/full.fmf`, e.g. `provision.hardware.cpu.processors` so `testing-farm` expects that there is a workable `fmf` tree even before the guest provisioning. I don't know how much of the fmf tree needs to be initialized at that point, nor do I know what relation does this step have with the copying of the fmf tree, so it seems rather risky.

Another issue is that this ties the testing to testing-farm + packit, but these tests should be able to be run downstream as well with `FedoraCI`, and it might break the reproducible script as well.

Reply:
Ok, thanks for the explanantion. It was worth a shot. Then let's go with the single `.fmf/version` in the root and everything else in `tools/fedora`.